x/deployment/simulation/operations.go

```diff
@@ -10,6 +9,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	"github.com/cosmos/cosmos-sdk/x/simulation"
+	"github.com/pkg/errors"
```

Comment:
nit: swap it with the `"github.com/cosmos/cosmos-sdk/baseapp"` import line for descending import specificity.